DATA/production/calib/tpc-laser-aggregator.sh

```diff
@@ -17,10 +30,11 @@ WORKFLOW="o2-dpl-raw-proxy $ARGS_ALL \
  --use-filtered-tracks --only-publish-on-eos \
  | o2-tpc-calib-pad-raw $ARGS_ALL \
   --configKeyValues \"TPCCalibPulser.FirstTimeBin=450;TPCCalibPulser.LastTimeBin=550;TPCCalibPulser.NbinsQtot=250;TPCCalibPulser.XminQtot=2;TPCCalibPulser.XmaxQtot=502;TPCCalibPulser.MinimumQtot=8;TPCCalibPulser.MinimumQmax=6;TPCCalibPulser.XminT0=450;TPCCalibPulser.XmaxT0=550;TPCCalibPulser.NbinsT0=400;keyval.output_dir=/dev/null\" \
- --lanes 36 \
- --publish-after-tfs 50 \
- --max-events 110
+ --lanes 1 \
```

Comment:
Do you want to make this also configurable?

Reply:
Indeed. I will implement this